RELEASENOTES.md

```diff
@@ -25,6 +25,7 @@ Now, you can set the checkbox called Use GhTokenWorkflow to allowing you to use
 
 ### New Settings
 - `keyVaultCodesignCertificateName`:  With this setting you can delegate the codesigning to an Azure Key Vault. This can be useful if your certificate has to be stored in a Hardware Security Module
+- `PRBuildTrigger`:  With this setting you can set which trigger to use for Pull Request Builds. By default Al-Go will use Pull_Request_Target
```

Comment:
Setting's name mismatch here.

Reply:
```suggestion
- `PRBuildTrigger`:  With this setting you can set which trigger to use for Pull Request Builds. By default Al-Go will use pull_request_target.
```

Reply:
My bad of adding to the same comment.
The first one still applies: the name of the new settings need to be the same in all the docs.